app/src/wrapper/awsSdkWrapper.js

```diff
@@ -0,0 +1,3 @@
+const AWS = require('aws-sdk');
```

Comment:
why is that required ?

Reply:
Because aws-sdk is saved as dev dependencies, all dependencies to aws-sdk in production code are marked as a error. That why this file is added to the esLint exclusion list. AWS-SDK is installed in AWS lambdas by default.